R/mod_tableGen.R

```diff
@@ -820,20 +830,9 @@ mod_tableGen_server <- function(input, output, session, datafile = reactive(NULL
       # Calculate totals for population set
       {total_for_code()}
       
+      tg_datalist <- list(ADAE = ae_data, ADSL = bds_data, POPDAT = {Rscript_use_preferred_pop_data()})
```

Comment:
I loaded tidyCDISC without any ADAE data and got the following error: 

![image](https://user-images.githubusercontent.com/17878953/190045123-af867b6c-8250-4f0a-99b0-865f44f2dcc5.png)

Reply:
I have updated the code to only include ADAE in the script if it was uploaded to the app.